Docs 5467 access management connected apps (#50)
mulesoft/docs-access-management: modules/ROOT/pages/connected-apps-overview.adoc

= Connected Apps
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> The Connected Apps feature provides a framework that enables an external application to integrate with Anypoint Platform using https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[APIs] via standard protocols, such as SAML, OAuth 2.0, and OpenID Connect.

Reviewer: ```suggestion
The Connected Apps feature provides a framework that enables an external application to integrate with Anypoint Platform using https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/[APIs] through standard protocols, such as SAML, OAuth 2.0, and OpenID Connect.
```

Connected apps help users delegate their access without sharing sensitive credentials or giving full control of their accounts to third parties.

> Actions taken by connected apps are auditable, and users can also revoke access at any time.

Author: ```suggestion
Actions taken by connected apps are audited, and users can also revoke access at any time.
Note that some products do not currently include client IDs in this release of the Connected Apps feature.
```

> The Connected Apps feature enables you to use secure authentication protocols and control an app's access to user data. Additionally, end users can authorize the app to access their Anypoint Platform data.

Reviewer: In the first line we are directly addressing the audience (good), but in the last line we use "end users" Who are end users? 
Is it You can control an app's access to user data and also authorize the app to access Anypoint Platform data?

Author: End users are non-devs and non-org admins. Regular ol' users who have Anypoint Platform credentials that they want to hand over to an app.

== Authentication

Anypoint Platform supports OAuth 2.0 and OpenID Connect to authorize apps to access data within Anypoint Platform.
OAuth (Open Authorization) is an open protocol that provides secure API authorization from applications in a simple and standardized way.
OAuth can authorize access to resources without revealing user credentials to apps.
OpenID Connect identifies the end user and obtains information to pass to OAuth 2.0 connected apps.

For more information, see https://oauth.net/2/[OAuth 2.0] and https://openid.net/connect/[OpenID Connect].

== Access and Authorization

You can use the Connected Apps feature to create a seamless authentication experience for end users. Connected apps address use cases for the following types of users:

* An xref:connected-apps-org-admin.adoc[organization administrator] can control how their organization’s data is used by whitelisting apps, revoking access, and disabling this feature for the entire organization.

> * A xref:connected-apps-developers.adoc#developers[developer] can register new and manage existing apps within their organization.

Reviewer: ```suggestion
* A xref:connected-apps-developers.adoc#developers[developer] can register new, and manage existing, apps within their organization.
```
